src/server.ts

```diff
@@ -4,7 +4,10 @@ import { startCronJobs } from "./cron";
 import dotenv from "dotenv-safe";
 
 console.log("Loading environment variables");
-dotenv.config();
+dotenv.config({
+  allowEmptyValues: true
```

Comment:
warum?
ich finde es eigentlich gut, wenn ich beim start der app auf fehlende Werte hingewiesen werde.
Dann sehe ich sofort, dass es ein problem gibt, anstatt dass ein fehler erst irgendwann später auftaucht.

Reply:
Wenn werte fehlen, crasht ja gleich das ganze node module, was ja unnötig ist, weil es ja grundsätzlich auch ohne einige der werte auskommt

Reply:
und der rest des codes is ja relativ robust, sodass eine fehlende env variable ordentlich behandelt wird